dts/bindings/regulator/cirrus,cp9314.yaml

```diff
@@ -39,3 +39,9 @@ properties:
     description: |
       Desired switched capacitor ratio set at initialization. This entry will overwrite
       the selection set by the PROG resistor.
+
+  cirrus,hw-i2c-lock:
```

Comment:
I think we might have had the same discussion for haptics but I believe you don't want "cirrus," préfixes (but that's probably for a follow-up PR, and I'm in my phone so can't easily check how much of an antipattern it is based on other bindings in-tree to see if it's worth making the change or living as-is)

Reply:
yes, we want for vendor specific properties. Zephyr is not consistent with this, but regulators are.